samples/core/src/main/java/io/oasp/gastronomy/restaurant/tablemanagement/service/impl/rest/TablemanagementRestServiceImpl.java

```diff
@@ -38,6 +40,7 @@ public void setTableManagement(Tablemanagement tableManagement) {
   @Override
   public TableEto getTable(String id) {
 
+    LoggerFactory.getLogger(TablemanagementRestServiceImpl.class).trace("impl test");
```

Comment:
Please follow our official log pattern (also automated in Eclipse via code template):
https://github.com/oasp/oasp4j/wiki/guide-logging#logger-access
However, IMHO "impl test" was not supposed to be committed at all...